### DISCUSSION TITLE
ros_omorobot_control please help me.. i use catkin_build

### DISCUSSION THREAD

**Discussion by kingseunghyun** [milvus-io/milvus#26316] [Q&A and General discussion]
Failed     << omo_r1mini_cartographer:cmake                        [ Exited with code 1 ]
Failed    <<< omo_r1mini_cartographer                              [ 2.2 seconds ]
[build] Summary: 9 of 10 packages succeeded.          
[build]   Ignored:   1 packages were skipped or are blacklisted.
[build]   Warnings:  None.                            
[build]   Abandoned: None.                            
[build]   Failed:    1 packages failed.               
[build] Runtime: 5.7 seconds total.    

***************************

Failed     << omo_r1mini_cartographer:cmake                        [ Exited with code 1 ]  help me 
im using catkin build but failed 
please come best programmer..........

**Comment by yhmo** [comment]
Sorry, we don't have any experience with catkin.
